packages/ruleset/src/functions/array-boundary.js

```diff
@@ -0,0 +1,75 @@
+const { validateSubschemas } = require('../utils');
+
+module.exports = function(schema, _opts, { path }) {
+  return validateSubschemas(schema, path, arrayBoundaryErrors, true, false);
+};
+
+const debugEnabled = false;
+function debug(msg) {
+  if (debugEnabled) {
+    console.log(msg);
+  }
+}
+
+const bypassFormats = {
+  minItems: [],
+  maxItems: []
+};
```

Comment:
I don't think there are any formats to exclude from this rule, this probably isn't needed.